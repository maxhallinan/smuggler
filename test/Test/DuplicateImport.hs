module Test.DuplicateImport where

import Data.Bool (not)
import Data.Bool (otherwise)

foo = not otherwise
