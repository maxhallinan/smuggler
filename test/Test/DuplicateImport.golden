module Test.DuplicateImport where

import Data.Bool (not, otherwise)

foo = not otherwise
